srcpkgs/python3-plotly/template

```diff
@@ -11,7 +11,7 @@ license="MIT"
 homepage="https://plot.ly/python/"
 changelog="https://github.com/plotly/plotly.py/blob/master/CHANGELOG.md"
 distfiles="${PYPI_SITE}/p/plotly/plotly-${version}.tar.gz"
-checksum=822eabe53997d5ebf23c77e1d1fcbf3bb6aa745eb05d532afd4b6f9a2e2ab02f
+checksum=859fdadbd86b5770ae2466e542b761b247d1c6b49daed765b95bb8c7063e7469
 
 do_check() {
```

Comment:
Please migrate this to `make_check=no # no tests are shipped` after `checksum`.

Reply:
Should `do_check` function be kept untouched?

Reply:
No, you can remove the whole function.

Reply:
Done